addons/project/report/project_report_views.xml

```diff
@@ -41,73 +41,27 @@
-                    <separator/>
-                    <filter name="filter_date_deadline" date="date_deadline"/>
-                    <filter name="filter_date_assign" date="date_assign"/>
-                    <filter name="filter_date_last_stage_update" date="date_last_stage_update"/>
-                    <separator/>
-                    <filter string="Archived" name="inactive" domain="[('active', '=', False)]"/>
-                    <group expand="0" string="Extended Filters">
-                        <field name="priority"/>
-                        <field name="company_id" groups="base.group_multi_company"/>
-                    </group>
-                    <group expand="1" string="Group By">
-                        <filter string="Stage" name="Stage" context="{'group_by': 'stage_id'}"/>
-                        <filter string="Personal Stage" name="personal_stage" context="{'group_by': 'personal_stage_type_ids'}"/>
-                        <filter string="Assignees" name="User" context="{'group_by': 'user_ids'}"/>
-                        <filter string="Ancestor Task" name="groupby_ancestor_task" context="{'group_by': 'ancestor_id'}" groups="project.group_subtask_project"/>
-                        <filter string="Milestone" name="milestone" context="{'group_by': 'milestone_id'}" groups="project.group_project_milestone"/>
-                        <filter string="Customer" name="Customer" context="{'group_by': 'partner_id'}"/>
-                        <filter string="Kanban State" name="kanban_state" context="{'group_by': 'state'}"/>
-                        <filter string="Deadline" name="deadline" context="{'group_by': 'date_deadline'}"/>
-                        <filter string="Creation Date" name="group_create_date" context="{'group_by': 'create_date'}"/>
-                    </group>
+                <search position="attributes">
+                    <attribute name="string">Tasks Analysis</attribute>
                 </search>
+                <field name="display_project_id" position="attributes">
+                    <attribute name="name">project_id</attribute>
+                </field>
+                <filter name="message_needaction" position="replace"/>
+                <filter name="activities_overdue" position="replace"/>
+                <filter name="activities_today" position="replace"/>
+                <filter name="activities_upcoming_all" position="replace"/>
             </field>
         </record>
 
        <record id="action_project_task_user_tree" model="ir.actions.act_window">
             <field name="name">Tasks Analysis</field>
             <field name="res_model">report.project.task.user</field>
             <field name="view_mode">graph,pivot</field>
+            <field name="domain">['|', ('my_activity_user_id', '=', False), ('my_activity_user_id', '=', uid)]</field>
```

Comment:
We have a field in the reporting with the next activity deadline, which needs to be duplicated for each user since view is generated on startup. This domain ensures that we only show the relevant record for the current user.
Anyway, I just realized that the filter has been removed since, so we can get rid of this